tests/everflow/everflow_test_utilities.py

```diff
@@ -789,12 +811,20 @@ def send_and_check_mirror_packets(self,
                 # mask off the DMAC and IP Checksum to verify the packet contents.
                 if self.mirror_type() == "egress":
                     mirror_packet_sent[packet.IP].ttl -= 1
-                    mirror_packet_sent[packet.Ether].src = setup[direction]["egress_router_mac"]
+                    if 't2' in setup['topo']:
+                        if duthost.facts['switch_type'] == "voq":
```

Comment:
is egress mirroring supported for voq ? if not we can remove this change and add it back when it is supported.

Reply:
egress mirroring is supported for voq